js_modules/dagster-ui/packages/ui-core/src/code-location/CodeLocationOverviewRoot.tsx

```diff
@@ -97,7 +97,7 @@ export const CodeLocationOverviewRoot = (props: Props) => {
           ) : null}
         </tbody>
       </Table>
-      <CodeLocationServerSection />
+      <CodeLocationServerSection locationName={repoAddress.location} />
```

Comment:
Could we do something like this: https://github.com/dagster-io/dagster/blob/d8d3c4800e88d98e5705f81717b73f02eae0d621/js_modules/dagster-ui/packages/ui-core/src/runs/RunMetricsDialog.oss.tsx#L3 ?